ui/component/livestreamLayout/view.jsx

```diff
@@ -2,7 +2,7 @@
 import 'scss/component/_swipeable-drawer.scss';
```

Comment:
`Discussion`
I haven't fully wrapped my head around Sass Partials yet, but if Dan's footprints is the correct example, I think we don't need the underscore here since it will not be combined into `all.scss` and should let it transpile separately for future lazy-loading?  Not 100% sure.